localstack/services/iam/iam_starter.py

```diff
@@ -255,18 +255,19 @@ def inline_policy_unapply_policy(self, backend):
 
     InlinePolicy.unapply_policy = inline_policy_unapply_policy
 
-    def instance_profile_create_from_cloudformation(resource_physical_name, cloudformation_json, region_name):
-        properties = cloudformation_json['Properties']
-
-        role_ids = [role.id for role in moto_iam_backend.roles.values() if role.name in properties['Roles']]
-
-        return moto_iam_backend.create_instance_profile(
-            name=resource_physical_name,
-            path=properties.get('Path', '/'),
-            role_ids=role_ids,
-        )
-
-    InstanceProfile.create_from_cloudformation_json = instance_profile_create_from_cloudformation
+    # TODO We already supported InstanceProfile resource in CFN
```

Comment:
I guess we could remove this code entirely, now that it is already supported elsewhere (not critical, though).